GVFS/GVFS.Build/GVFS.props

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup Label="Parameters">
     <GVFSVersion>0.2.173.2</GVFSVersion>
-    <GitPackageVersion>2.20181107.1</GitPackageVersion>
+    <GitPackageVersion>2.20181106.5-pr</GitPackageVersion>

```

Comment:
I talked this over with Ben and want to change the indicator to be updatedSkipWorktreeBits since it is not just the `reset --mixed` that we care about it is when the index changes and the skip-worktree bits could have been changed.  I also need to update the description since the implementation has changed